tests/unit/amo/helpers.js

```diff
@@ -66,17 +66,30 @@ export const signedInApiState = Object.freeze({
   clientApp: 'firefox',
 });
 
-export function dispatchSignInActions({
+export function dispatchAnonymousActions({
```

Comment:
I think `dispatchClientMetaData` would more accurately express what this is doing. It's simulating how the client will set its lang and user agent.